### ISSUE TITLE
Thinks colorama.Fore.COLORNAME are integers -- they are strings.

### ISSUE THREAD

**Issue by gvanrossum** [astral-sh/ty#3475]
### Summary

I have this code in file _tt.py_:
```py
from colorama import Fore
print(Fore.RED + "This text is red" + Fore.RESET)
```
Assume `colorama` is available in the active _venv_. It runs fine and prints a line in red.

Running `ty check tt.py` however gives this incorrect error:
```
warning[unsupported-operator]: Unsupported `+` operation
 --> tt.py:2:7
  |
2 | print(Fore.RED + "This text is red" + Fore.RESET)
  |       --------^^^------------------
  |       |          |
  |       |          Has type `Literal["This text is red"]`
  |       Has type `int`
  |
```

Now, from the source it looks as if `colorama` constructs `Fore` using a bit of a trick. But _pyright_ does not complain here, and I use code this many times, so until this is fixed I cannot use _ty_ in my project.

**Comment by charliermarsh** [comment]
This can be resolved by installing the colorama stubs ([`types-colorama`](https://pypi.org/project/types-colorama/)). We're considered vendoring stubs for third-party packages [here](https://github.com/astral-sh/ty/issues/1104).

**Comment by charliermarsh** [comment]
For posterity, colorama dynamically converts these here: https://github.com/tartley/colorama/blob/3de9f013df4b470069d03d250224062e8cf15c49/colorama/ansi.py#L33. The stubs type them here: https://github.com/python/typeshed/blob/098f30ecd13f56c4cef95ed47afe281c1a317dbe/stubs/colorama/colorama/ansi.pyi#L20-L39.

**Comment by charliermarsh** [comment]
We should either vendor the stubs, or alert the user that they need to install them; we're tracking that in https://github.com/astral-sh/ty/issues/1104.

**Comment by gvanrossum** [comment]
> We should either vendor the stubs, or alert the user that they need to install them; we're tracking that in [#1104](https://github.com/astral-sh/ty/issues/1104).

Thanks! That works. I am experiencing an overwhelming number of other issues. I will try to file a few more.
